src/test/java/core/basesyntax/strategy/impl/OperationStrategyImplTest.java

```diff
@@ -0,0 +1,56 @@
+package core.basesyntax.strategy.impl;
+
+import static core.basesyntax.model.FruitTransaction.Operation.BALANCE;
+import static core.basesyntax.model.FruitTransaction.Operation.PURCHASE;
+import static core.basesyntax.model.FruitTransaction.Operation.RETURN;
+import static core.basesyntax.model.FruitTransaction.Operation.SUPPLY;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.Map;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class OperationStrategyImplTest {
+    private OperationStrategy operationStrategy;
+
+    @BeforeEach
+    void setUp() {
+        Map<FruitTransaction.Operation, OperationHandler> map = Map.of(
+                BALANCE, new BalanceOperationHandler(),
+                SUPPLY, new SupplyOperationHandler(),
+                PURCHASE, new PurchaseOperationHandler(),
+                RETURN, new ReturnOperationHandler());
+        operationStrategy = new OperationStrategyImpl(map);
+    }
+
+    @Test
+    void getOperationStrategy_balanceOperation_ok() {
+        OperationHandler expacted = new BalanceOperationHandler();
```

Comment:
```suggestion
        OperationHandler expected = new BalanceOperationHandler();
```

Fix typo in all places.